aten/src/ATen/native/native_functions.yaml

```diff
@@ -4256,6 +4256,10 @@
   use_c10_dispatcher: full
   variants: function
 
+- func: choose_qparams_optimized(Tensor input, int n, int n_bins, float ratio, float xmin, float xmax, int bit_rate) -> (float, float)
```

Comment:
NIT: Can we call this bit_width instead of bit_rate?
Also, do we distinguish between fp16 and fp32 outputs?

Reply:
This function returns the optimized xmin and xmax values currently. The calculation of scale and zero point happens at the callsite,.